dot/parachain/availability-store/availability_store.go

```diff
@@ -170,7 +175,10 @@ func uint32ToBytes(value uint32) []byte {
 // Run runs the availability store subsystem
 func (av *AvailabilityStoreSubsystem) Run(ctx context.Context, OverseerToSubsystem chan any,
 	SubsystemToOverseer chan any) error {
-	av.processMessages()
+
+	av.wg.Add(2)
```

Comment:
I don't think there is a need for the waitGroup here.

Reply:
a goroutine starts after this line. It is just a nice practice to make sure that the goroutine has ended when we run the Stop() function.